python/python-core/src/main/python/_timefold_java_interop.py

```diff
@@ -80,20 +80,30 @@ def init(*args, path: list[str] = None, include_timefold_jars: bool = True) -> N
 
     if jpype.isJVMStarted():  # noqa
         raise RuntimeError('JVM already started. Maybe call init before timefold.solver.types imports?')
+
     if path is None:
         include_timefold_jars = True
         path = []
     if include_timefold_jars:
         path = path + extract_timefold_jars()
     if len(args) == 0:
-        args = [jpype.getDefaultJVMPath()]
+        args = [get_default_jvm_path()]
     init(*args, path=path, include_translator_jars=False)
 
     from ai.timefold.solver.python.logging import PythonDelegateAppender
     PythonDelegateAppender.setLogEventConsumer(PythonConsumer(forward_logging_events))
     update_log_level()
 
 
+def get_default_jvm_path():
```

Comment:
Can use `path_supplier=jpype.getDefaultJVMPath` as a parameter to make it unit testable, and do `path_supplier()` in the `try` block.